src/PythonMigrationViewExtension/PythonMigrationViewExtension.cs

```diff
@@ -71,8 +71,8 @@ public void Loaded(ViewLoadedParams p)
 
         private void DisplayIronPythonDialog()
         {
-            // we only want to create the dialog ones for each graph per Dynamo session
-            if (DialogTracker.ContainsKey(CurrentWorkspace.Guid))
+            // we only want to create the dialog ones for each graph per Dynamo session, if the global setting is not disabled
+            if (DialogTracker.ContainsKey(CurrentWorkspace.Guid) || DynamoViewModel.IsIronPythonDialogDisabled)
```

Comment:
ah.. Not really related to this task but this line might be the reason why we have bug for dyf everytime we open..

Reply:
yes, tracking guid and not function id?

Reply:
@mjkkirschner yup

Reply:
Do you mean this bug?
https://jira.autodesk.com/browse/DYN-2936

Reply:
Nope this one: https://jira.autodesk.com/browse/DYN-2937. Anyway, I think out of scope..